docker-compose.yml

```diff
@@ -2,8 +2,6 @@ db:
   image: mongo:latest
 web:
   build: .
-  working_dir: /vagrant/oratory/
-  command: supervisor server.js
```

Comment:
I opted to have them there instead of the `Dockerfile` so that we could change the running command without having to rebuild the image.  It's really the same thing though so I'm all for just moving that into the `Dockerfile`